server/middlewares/validations/AuthValidation.js

```diff
@@ -118,6 +118,39 @@ class UserInputValidation {
       errors: validation.errors.all()
     });
   }
+
+  /**
+   * validate user input on email Verification
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {func} next
+   *
+   * @return {void}
+   */
+  static emailInputValidation(req, res, next) {
+    const { email } = req.body.user;
+    const validation = new Validator(
+      {
+        email
+      },
+      {
+        email: 'required|string|email',
```

Comment:
No need for `string` as part of validation, email type covers for that

Reply:
I have been removed.